vmm/src/lib.rs

```diff
@@ -212,6 +212,23 @@ pub enum VmmActionError {
     VsockConfig(ErrorKind, VsockError),
 }
 
+// It's convenient to turn DriveErrors into VmmActionErrors directly.
+impl std::convert::From<DriveError> for VmmActionError {
+    fn from(e: DriveError) -> Self {
+        let kind = match e {
+            DriveError::CannotOpenBlockDevice => ErrorKind::User,
```

Comment:
If all of them match to `User`, why not `return ErrorKind::User` directly? Or use just a match arm with `_`?

Reply:
Definitely don't want to do that. Need to be explicit so any added Error will result in compilation error unless this is implemented for it as well. A catch all will indiscriminately.. well.. catch all.